docs/api-reference/webgl/buffer.md

```diff
@@ -64,7 +64,7 @@ buffer.subData({offset: 20, data: new Float32Array([1, 2, 3])});
 Getting data from a buffer (WebGL2)
 ```js
 const buffer = ...;
-const data = buffer.getSubData({offset: 20, size: 10});
+const data = buffer.getData({srcByteOffset: 32, length: 5});
```

Comment:
Maybe add a couple of `getData` parameter variations and add `// comments` in front of each `getData` example explaining what effect these "cryptic" parameters have.